services/docker-compose.yml

```diff
@@ -168,17 +168,21 @@ services:
       - computational_services_subnet
 
   rabbit:
-    image: rabbitmq:3.7.17-management
+    image: rabbitmq:3.8.0-management
```

Comment:
There has been an issue with aiopika and rabbitmq. The server would have open/closed connections indefinitely after some time. But I think that was not related to rabbit itself. Just go ahead. I the server starts to behave weird (a flood of log/error messages essentially) we can have closer look